src/olympia/versions/tasks.py

```diff
@@ -333,11 +333,14 @@ def duplicate_addon_version_for_rollback(*, version_pk, new_version_number, user
             details={
                 # The comment is not translated on purpose, to behave like regular human
                 # approval does.
-                'comments': f'{text} by re-publishing as "{new_version_number}", '
+                'comments': f'{text} by re-publishing as "{new_version_number}" '
                 'successfull!\n'
```

Comment:
(Existing) typo ?
```suggestion
                'successful!\n'
```

Also appears in test below